exonum-java-binding-core/rust/integration_tests/tests/utils_errors.rs

```diff
@@ -22,8 +22,16 @@ const ARITHMETIC_EXCEPTION_CLASS_FQN: &str = "java.lang.ArithmeticException";
 const CUSTOM_EXCEPTION_MESSAGE: &str = "Test exception message";
 
 lazy_static! {
-    static ref VM: Arc<JavaVM> = create_vm_for_tests();
-    pub static ref EXECUTOR: MainExecutor = MainExecutor::new(VM.clone());
+    static ref VM: Arc<JavaVM> = create_vm_for_tests_with_fake_classes();
+    pub static ref EXECUTOR: MainExecutor = {
+        let ex = MainExecutor::new(VM.clone());
+        // JNI_OnLoad() is not called for these tests
+        ex.with_attached(|env|{
+            jni_cache::cache_methods(env);
```

Comment:
It seems `JNI_OnLoad()` is not called for `utils_errors` family of tests, so I still see no better solution here.